### ISSUE TITLE
Deadlines for Veronica

### ISSUE THREAD

**Issue by vocatwk** [vocatwk/CodeU-Nemo#76]
Below are some of the features that I'd like to have done by the showcase:

- add/remove users (By the end of this weekend) 
- access control for conversations (By Wednesday July 4th)
- Messaging other users privately (By Saturday July 7th)
- editing/deleting messages (By Sunday July 8th)
- replying to a message/ liking a message (By thursday July 12th)
- uploading profile pictures (By Saturday July 14th)
- sharing files(images and textfile to start with) (By sunday 15th)

**Comment by vocatwk** [comment]
@yebrahim @lealgamb @jasminechau2 I'm not sure whether to keep working on access control(allowing only the creator of a conversation to add/remove people) or implement one of the reach goals above or work on implementing the bootstrap design. What do you guys think would be more relevant for us?

**Comment by yebrahim** [comment]
That’s really up to you of course.

Since the focus of this project is learning, I would encourage you to pick an area you’re least familiar with, and that you want to play around with a little more. Uploading pictures, sending files, maybe emojis or some more frontend work can be examples of that.

**Comment by jasminechau2** [comment]
I agree with Yasser. If you want a change from what you are doing right now it would be good to try to do one of the stretch goals.

**Comment by vocatwk** [comment]
Yeah you're right, Thanks guys. I'm going to give profile pictures a try.
